File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/before/EmptyBeforeResponseHandlerImpl.java
package ua.com.lavi.komock.engine.handler.before;

import org.jetbrains.annotations.NotNull;
import ua.com.lavi.komock.engine.model.Request;
import ua.com.lavi.komock.engine.model.Response;

public class EmptyBeforeResponseHandlerImpl implements BeforeResponseHandler {

    @Override
    public void handle(@NotNull Request request, @NotNull Response response) {
        // nothing to do
    }
}

File: komock-core/src/main/kotlin/ua/com/lavi/komock/engine/handler/before/EmptyBeforeResponseHandlerImpl.kt
package ua.com.lavi.komock.engine.handler.before

import ua.com.lavi.komock.engine.model.Request
import ua.com.lavi.komock.engine.model.Response

/**
 * Created by Oleksandr Loushkin
 */

class EmptyBeforeResponseHandlerImpl : BeforeResponseHandler {

    override fun handle(request: Request, response: Response) {
        // nothing to do
    }
}
